src/utils/config.ts

```diff
@@ -48,3 +48,22 @@ export const config = createConfig({
   batch: { multicall: true },
   connectors,
 });
+
+// Random test address
+export const USER_TEST_ADDRESS = '0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045';
```

Comment:
Ideally we should try not to mix test code with main code, to prevent future bugs and keep code clean and maintainable

Reply:
https://github.com/0xShishigami/feo-web3-react-boilerplate/pull/6/commits/2d76e29c0c4caf48b422ac350767bdba1b2b3105